tests/system/action/motion/test_create.py

```diff
@@ -401,3 +434,34 @@ def test_create_permission_lead_motion(self) -> None:
             },
         )
         self.assert_status_code(response, 200)
+        self.assert_model_exists("motion/4", {"block_id": 13, "category_id": 12})
+
+    def test_create_lead_motion_extra_category_id(self) -> None:
```

Comment:
name proposal: `test_create_amendment_no_perms_category_id`.

Maybe also test the `block_id` in a second test.

Reply:
Add a test and renamed the test.